journalism_workflow/static/journalism_workflow/copy_editor/js/controllers.js

```diff
@@ -0,0 +1,30 @@
+(function () {
+  'use strict';
+
+  angular
+    .module('journalism_workflow.copy_editor.controllers')
+    .controller('CopyEditorController', CopyEditorController)
+
+  CopyEditorController.$inject = ['$scope', 'orchestraService'];
+
+  function CopyEditorController($scope, orchestraService) {
+    var vm = $scope;
+
+    // Store the article text document URL for easier summary later
+    var documentCreationStep = orchestraService.taskUtils.findPrerequisite(
+      vm.taskAssignment, 'document_creation')
+    vm.taskAssignment.task.data.articleDocument = documentCreationStep.task.data.articleURL;
+
+    // Set up the photos for captioning
+    vm.taskAssignment.task.data.photos = [];
+    var photoAdjustStep = orchestraService.taskUtils.findPrerequisite(
+      vm.taskAssignment, 'photo_adjust');
```

Comment:
my feedback isn't that we should do more complicated stuff with PIL.  my feedback is that showing the greyscale photos is not what you'd do in reality---the image processing is going to generate lots of versions of each image, and the copy editor is going to look at originals to generate captions, etc.

BUT, it's more pedagogically interesting to show the greyscale ones, and i'm alright with that.